packages/edit-site/src/components/sidebar-edit-mode/global-styles-sidebar.js

```diff
@@ -69,11 +77,14 @@ export default function GlobalStylesSidebar() {
 							label={ __( 'Style Book' ) }
 							isPressed={ isStyleBookOpened }
 							disabled={ shouldClearCanvasContainerView }
-							onClick={ () =>
+							onClick={ () => {
+								setIsListViewOpened(
+									isStyleBookOpened && showListViewByDefault
```

Comment:
This PR is working well for the style book for me. Thank you!

I was wondering if, while we're at it, we could give the same treatment to the revisions panel. 

I think, along with the changes in this PR, the following would do it:

```diff
diff --git a/packages/edit-site/src/components/global-styles/ui.js b/packages/edit-site/src/components/global-styles/ui.js
index 218665c682..bd73785e44 100644
--- a/packages/edit-site/src/components/global-styles/ui.js
+++ b/packages/edit-site/src/components/global-styles/ui.js
@@ -47,6 +47,7 @@ const { Slot: GlobalStylesMenuSlot, Fill: GlobalStylesMenuFill } =
 
 function GlobalStylesActionMenu() {
 	const { toggle } = useDispatch( preferencesStore );
+	const { setIsListViewOpened } = useDispatch( editSiteStore );
 	const { canEditCSS, revisionsCount } = useSelect( ( select ) => {
 		const { getEntityRecord, __experimentalGetCurrentGlobalStylesId } =
 			select( coreStore );
@@ -71,6 +72,7 @@ function GlobalStylesActionMenu() {
 	);
 	const loadCustomCSS = () => goTo( '/css' );
 	const loadRevisions = () => {
+		setIsListViewOpened( false );
 		goTo( '/revisions' );
 		setEditorCanvasContainerView( 'global-styles-revisions' );
 	};

```

If it ends up causing other problems, then don't worry about it :)

Reply:
Thanks for looking into this. Sure, I will add those changes.